src/peripherals/LentMycReader.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
+import {ILentMyc} from "../interfaces/ILentMyc.sol";
+
+/**
+ * @title MYC Lending contract reader
+ * @author Dospore
+ */
+contract LentMycReader {
+    using FixedPointMathLib for uint256;
+
+    /// @notice External function to access internal _getCurrentCycleInfo
+    function getCurrentCycleInfo(address lentMyc_, uint256 ethRewards) external view returns (uint256[] memory) {
+        return _getCurrentCycleInfo(lentMyc_, ethRewards);
+    }
+
+    /// @notice Get the current cycles reward reward and info given an eth reward amount
+    /// @dev Dospore
+    /// @param lentMyc_ address of lentMyc
+    /// @param ethRewards amount of eth that will be rewarded to current cycle stakers
+    /// @return Current cycles reward info based on the given eth rewards amount
```

Comment:
Change natspecs to use `/** */` comments please :pleading_face: 

Also full sentences please. Sorry for being annoying, just keeping styling consistent with the main contracts.

Reply:
I added the natspecs and will update the sentences. But what do you mean by full sentences just like more explanation?